lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps a certain token amount for a minimum of another token amount.
+
+  ## Arguments
+    * `account` - An elrond account (as described in lib/account)
+    * `pair` -  An elrond pair (as described in lib/pair)
+    * `token_in` - First token to be swapped
+    * `value_in` - Fixed amount of first token to be swapped
```

Comment:
Fix value amount for input token. 
This value ammount will be transferred on Pair SC in order to perform swap.